i2c-tools.yaml

```diff
@@ -16,8 +19,9 @@ environment:
       - ca-certificates-bundle
       - linux-headers
       - openssf-compiler-options
-      - py3.12-setuptools
-      - python3-dev
+      - py${{vars.py-version}}-build-base
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-dev
```

Comment:
We want to avoid this one, because it adds `/usr/bin/python3` to the build environment. Instead, you can just change `py${{vars.py-version}}-build-base` to `py${{vars.py-version}}-build-base-dev` (adding `-dev`), and you'll get the headers for `python3.13` w/o the `/usr/bin/python3` symlink.